src/compass/utils/schemas/cslc_s1.yaml

```diff
@@ -11,7 +11,7 @@ runconfig:
            # Required. List of orbit (EOF) files
            orbit_file_path: list(str(), min=1)
            # List of (unique) burst IDs to process
-           burst_id: list(str(), min=1, required=True)
+           burst_id: str(required=True)
```

Comment:
This changes the behavior of the RSLC workflow. Is it intentional? I am not sure if I missed something in the conversion.

Reply:
Yes, this is intentional. Both the workflows will process one burst at the time. I modified the comments to be consistent.